server/services/visitService.ts

```diff
@@ -206,6 +207,13 @@ export default class VisitService {
     }
   }
 
+  async getNotificationCount(username: string, prisonId: string): Promise<NotificationCount> {
```

Comment:
I've added this to `VisitService` for now but as we get more functionality for reviewing bookings it might warrant splitting to a different service.